extension/src/cli/dvc/constants.ts

```diff
@@ -1,4 +1,4 @@
-export const MIN_CLI_VERSION = '2.24.0'
+export const MIN_CLI_VERSION = '2.0.0'
```

Comment:
[C] Cannot merge and ship before new version drops (https://github.com/iterative/dvc/pull/8282)